Dockerfile

```diff
@@ -7,6 +7,9 @@ FROM aligent/shopify-cli:latest
 ARG USER_ID
 ARG GROUP_ID
 
+RUN groupadd -g "$GROUP_ID" shopifygroup && \
```

Comment:
I think you need this (or at least part of this) in the entrypoint because it needs to user the current user's group and user id. Which it will only have at run time

Reply:
Thanks @TheOrangePuff, I added a catcher in entrypoint.sh.